modules/schema/src/test/java/org/apache/ignite/internal/schema/RowTest.java

```diff
@@ -697,6 +697,10 @@ private void checkValues(SchemaDescriptor schema, Object... vals) {
             }
         }
 
+        BinaryRow row0 = asm.build();
+
+        assert vals.length > 0 && row0.bytes().length > 0;
```

Comment:
this complex test is enough as i can see.

Reply:
As you said, this test already complex. Don't see any reason to increase its complexity

Reply:
this additional check, checks that serializer rewind data store before send in into pipe, otherwise :
![image](https://user-images.githubusercontent.com/1107510/154414751-63f3e2ea-87bf-4c91-ac82-c62501753b63.png)
fnd deeper into : CommandUtils.rowsToBytes(rows); will give null row

Reply:
ok, now it`s under : ExpandableByteBufTest#allTypesDirectOrder